benchmarks/registry.js

```diff
@@ -16,15 +16,19 @@ function setupRegistrySuite(suite) {
 	labelSetups.forEach(({ name, counts }) => {
 		suite.add(
 			`getMetricsAsJSON#${name}`,
-			(client, registry) => registry.getMetricsAsJSON(),
+			async (client, registry) => await registry.getMetricsAsJSON(),
 			{ setup: setup(counts) },
 		);
 	});
 
 	labelSetups.forEach(({ name, counts }) => {
-		suite.add(`metrics#${name}`, (client, registry) => registry.metrics(), {
-			setup: setup(counts),
-		});
+		suite.add(
+			`metrics#${name}`,
+			async (client, registry) => await registry.metrics(),
```

Comment:
this doesn't need to change does it? Just returning the promise seems like it'd work?